src/instance.ts

```diff
@@ -214,33 +218,28 @@ export class VMInstance {
     return this.do_query_chain(request).ptr;
   }
 
-  abort(message_ptr: number, file_ptr: number, line: number, column: number) {
+  abort(message_ptr: number) {
```

Comment:
These parameters are never set

Reply:
I’m very certain `file_ptr` is indeed being supplied, and that `line` & `column` are a part of the `file_ptr`